<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta name="google-site-verification" content=""> <meta http-equiv="Permissions-Policy" content="interest-cohort=()"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Publications | Shubhashis Roy Dipta</title> <meta name="author" content="Shubhashis Roy Dipta"> <meta name="description" content="Personal site of Shubhashis Roy Dipta "> <meta name="keywords" content="shubhashis-roy-dipta, shubhashis-roy, dipta, roy-dipta, shubhashis-dipta, dipta007"> <meta property="og:site_name" content="Shubhashis Roy Dipta"> <meta property="og:type" content="website"> <meta property="og:title" content="Shubhashis Roy Dipta | Publications"> <meta property="og:url" content="https://roydipta.com/publications/"> <meta property="og:description" content="Personal site of Shubhashis Roy Dipta "> <meta property="og:image" content="/assets/img/social_preview.png"> <meta property="og:locale" content="en"> <meta name="twitter:card" content="summary"> <meta name="twitter:title" content="Publications"> <meta name="twitter:description" content="Personal site of Shubhashis Roy Dipta "> <meta name="twitter:image" content="/assets/img/social_preview.png"> <meta name="twitter:site" content="@iamdipta007"> <meta name="twitter:creator" content="@iamdipta007"> <script type="application/ld+json">
    {
        "author":
        {
            "@type": "Person",
            "name": "Shubhashis  Roy Dipta"
        },
        "url": "https://roydipta.com/publications/",
        "@type": "WebSite",
        "description": "Personal site of Shubhashis Roy Dipta
",
        "headline": "Publications",
        "sameAs": ["https://scholar.google.com/citations?user=j2ElsNIAAAAJ", "https://github.com/dipta007", "https://www.linkedin.com/in/dipta007", "https://twitter.com/iamdipta007", "https://youtube.com/@roydipta"],
        "name": "Shubhashis  Roy Dipta",
        "@context": "https://schema.org"
    }
    </script> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?16404ec2cd2689e8d0f38f73fe0d38f9"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?19f3075a2d19613090fe9e16b564e1fe" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%8F%A0&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://roydipta.com/publications/"> </head> <body class="fixed-top-nav sticky-bottom-footer"> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/">Shubhashis¬†<span class="font-weight-bold">Roy Dipta</span></a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">Projects</a> </li> <li class="nav-item "> <a class="nav-link" href="https://notes.roydipta.com/?utm_source=roydipta.com&amp;utm_medium=header" rel="external nofollow noopener" target="_blank">Notes</a> </li> <li class="nav-item "> <a class="nav-link" href="https://blog.roydipta.com/?utm_source=roydipta.com&amp;utm_medium=header" rel="external nofollow noopener" target="_blank">Blog</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Misc.</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/awards/">Awards üèÜ</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/certificates/">Certificates üìú</a> </div> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications <a href="https://scholar.google.com/citations?user=j2ElsNIAAAAJ&amp;hl=en" target="_blank" rel="noopener noreferrer" class="float-right"> <i class="ai ai-google-scholar"></i> </a> </h1> <p class="post-description"></p> </header> <article> <div class="publications"> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#ED1C24"><a href="https://aclanthology.org/" rel="external nofollow noopener" target="_blank">ACL</a></abbr></div> <div id="roy-dipta-etal-2023-semantically" class="col-sm-8"> <div class="title">Semantically-informed Hierarchical Event Modeling</div> <div class="author"> <b><u>Shubhashis Roy Dipta</u></b>,¬†<a href="https://scholar.google.com/citations?user=ZdhvnD8AAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">Mehdi Rezaee</a>,¬†and¬†<a href="https://redirect.cs.umbc.edu/~ferraro/" rel="external nofollow noopener" target="_blank">Francis Ferraro</a> </div> <div class="periodical"> <em>In Proceedings of the 12th Joint Conference on Lexical and Computational Semantics (*SEM 2023)</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2212.10547" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://aclanthology.org/2023.starsem-1.31" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://aclanthology.org/2023.starsem-1.31.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> <a href="https://github.com/dipta007/SHEM" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <span class="__dimensions_badge_embed__" data-doi="10.18653/v1/2023.starsem-1.31" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Prior work has shown that coupling sequential latent variable models with semantic ontological knowledge can improve the representational capabilities of event modeling approaches. In this work, we present a novel, doubly hierarchical, semi-supervised event modeling framework that provides structural hierarchy while also accounting for ontological hierarchy. Our approach consistsof multiple layers of structured latent variables, where each successive layer compresses and abstracts the previous layers. We guide this compression through the injection of structured ontological knowledge that is defined at the type level of events: importantly, our model allows for partial injection of semantic knowledge and it does not depend on observing instances at any particular level of the semantic ontology. Across two different datasets and four different evaluation metrics, we demonstrate that our approach is able to out-perform the previous state-of-the-art approaches by up to 8.5%, demonstrating the benefits of structured and semantic hierarchical knowledge for event modeling.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">roy-dipta-etal-2023-semantically</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Semantically-informed Hierarchical Event Modeling}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Roy Dipta, Shubhashis and Rezaee, Mehdi and Ferraro, Francis}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 12th Joint Conference on Lexical and Computational Semantics (*SEM 2023)}</span><span class="p">,</span>
  <span class="na">month</span> <span class="p">=</span> <span class="nv">jul</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">address</span> <span class="p">=</span> <span class="s">{Toronto, Canada}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Association for Computational Linguistics}</span><span class="p">,</span>
  <span class="na">url</span> <span class="p">=</span> <span class="s">{https://aclanthology.org/2023.starsem-1.31}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.18653/v1/2023.starsem-1.31}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{353--369}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#B31B1B"><a href="https://arxiv.org/" rel="external nofollow noopener" target="_blank">arXiv</a></abbr></div> <div id="saha2023seebel" class="col-sm-8"> <div class="title">SeeBel: Seeing is Believing</div> <div class="author"> <a href="https://sourajitcs.github.io/" rel="external nofollow noopener" target="_blank">Sourajit Saha</a>,¬†and¬†<b><u>Shubhashis Roy Dipta</u></b> </div> <div class="periodical"> <em>arXiv preprint</em>, Jul 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2312.10933" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://arxiv.org/pdf/2312.10933.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> <span class="__dimensions_badge_embed__" data-pmid="" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Semantic Segmentation is a significant research field in Computer Vision. Despite being a widely studied subject area, many visualization tools do not exist that capture segmentation quality and dataset statistics such as a class imbalance in the same view. While the significance of discovering and introspecting the correlation between dataset statistics and AI model performance for dense prediction computer vision tasks such as semantic segmentation is well established in the computer vision literature, to the best of our knowledge, no visualization tools have been proposed to view and analyze the aforementioned tasks. Our project aims to bridge this gap by proposing three visualizations that enable users to compare dataset statistics and AI performance for segmenting all images, a single image in the dataset, explore the AI model‚Äôs attention on image regions once trained and browse the quality of masks predicted by AI for any selected (by user) number of objects under the same tool. Our project tries to further increase the interpretability of the trained AI model for segmentation by visualizing its image attention weights. For visualization, we use Scatterplot and Heatmap to encode correlation and features, respectively. We further propose to conduct surveys on real users to study the efficacy of our visualization tool in computer vision and AI domain.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">saha2023seebel</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SeeBel: Seeing is Believing}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Saha, Sourajit and Roy Dipta, Shubhashis}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{arXiv preprint}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{arXiv}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#FDB515"><a href="https://www.springer.com/" rel="external nofollow noopener" target="_blank">Springer</a></abbr></div> <div id="islam2022methevo" class="col-sm-8"> <div class="title">MethEvo: an accurate evolutionary information-based methylation site predictor</div> <div class="author"> Sadia Islam,¬†Shafayat Bin Shabbir Mugdha,¬†<b><u>Shubhashis Roy Dipta</u></b>,¬†MD Easin Arafat,¬†<a href="https://scholar.google.com/citations?user=_D9NsFMAAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">Swakkhar Shatabda</a>,¬†<a href="https://www.unsw.edu.au/staff/hamid-alinejad-rokny" rel="external nofollow noopener" target="_blank">Hamid Alinejad-Rokny</a>,¬†and¬†<a href="https://cs.camden.rutgers.edu/faculty-staff/iman-dehzangi-ph-d/" rel="external nofollow noopener" target="_blank">Iman Dehzangi</a> </div> <div class="periodical"> <em>Neural Computing and Applications</em>, Jul 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s00521-022-07738-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <span class="__dimensions_badge_embed__" data-doi="10.1007/s00521-022-07738-9" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">islam2022methevo</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{MethEvo: an accurate evolutionary information-based methylation site predictor}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Islam, Sadia and Mugdha, Shafayat Bin Shabbir and Dipta, Shubhashis Roy and Arafat, MD Easin and Shatabda, Swakkhar and Alinejad-Rokny, Hamid and Dehzangi, Iman}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Neural Computing and Applications}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--12}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s00521-022-07738-9}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#5F497A"><a href="https://www.mdpi.com/journal/genes" rel="external nofollow noopener" target="_blank">Genes</a></abbr></div> <div id="arafat2020accurately" class="col-sm-8"> <div class="title">Accurately predicting glutarylation sites using sequential bi-peptide-based evolutionary features</div> <div class="author"> Md Easin Arafat,¬†Md Wakil Ahmad,¬†SM Shovan,¬†<a href="https://cs.camden.rutgers.edu/faculty-staff/iman-dehzangi-ph-d/" rel="external nofollow noopener" target="_blank">Abdollah Dehzangi</a>,¬†<b><u>Shubhashis Roy Dipta</u></b>,¬†Md Al Mehedi Hasan,¬†<a href="https://www.wilkes.edu/campus-directory/ghazaleh.taherzadeh.aspx" rel="external nofollow noopener" target="_blank">Ghazaleh Taherzadeh</a>,¬†<a href="https://scholar.google.com/citations?user=_D9NsFMAAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">Swakkhar Shatabda</a>,¬†and¬†<a href="https://scholar.google.com/citations?user=RVEFII8AAAAJ&amp;hl=en&amp;oi=ao" rel="external nofollow noopener" target="_blank">Alok Sharma</a> </div> <div class="periodical"> <em>Genes</em>, Jul 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.mdpi.com/2073-4425/11/9/1023" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <span class="__dimensions_badge_embed__" data-doi="10.3390/genes11091023" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Post Translational Modification (PTM) is defined as the alteration of protein sequence upon interaction with different macromolecules after the translation process. Glutarylation is considered one of the most important PTMs, which is associated with a wide range of cellular functioning, including metabolism, translation, and specified separate subcellular localizations. During the past few years, a wide range of computational approaches has been proposed to predict Glutarylation sites. However, despite all the efforts that have been made so far, the prediction performance of the Glutarylation sites has remained limited. One of the main challenges to tackle this problem is to extract features with significant discriminatory information. To address this issue, we propose a new machine learning method called BiPepGlut using the concept of a bi-peptide-based evolutionary method for feature extraction. To build this model, we also use the Extra-Trees (ET) classifier for the classification purpose, which, to the best of our knowledge, has never been used for this task. Our results demonstrate BiPepGlut is able to significantly outperform previously proposed models to tackle this problem. BiPepGlut achieves 92.0%, 84.8%, 95.6%, 0.82, and 0.88 in accuracy, sensitivity, specificity, Matthew‚Äôs Correlation Coefficient, and F1-score, respectively. BiPepGlut is implemented as a publicly available online predictor.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">arafat2020accurately</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Accurately predicting glutarylation sites using sequential bi-peptide-based evolutionary features}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Arafat, Md Easin and Ahmad, Md Wakil and Shovan, SM and Dehzangi, Abdollah and Dipta, Shubhashis Roy and Hasan, Md Al Mehedi and Taherzadeh, Ghazaleh and Shatabda, Swakkhar and Sharma, Alok}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Genes}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{11}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1023}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{MDPI}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.3390/genes11091023}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#0B6EA7"><a href="https://www.ieee.org/" rel="external nofollow noopener" target="_blank">IEEE Access</a></abbr></div> <div id="ahmad2020mal" class="col-sm-8"> <div class="title">Mal-light: Enhancing lysine malonylation sites prediction problem using evolutionary-based features</div> <div class="author"> Md Wakil Ahmad,¬†Md Easin Arafat,¬†<a href="https://www.wilkes.edu/campus-directory/ghazaleh.taherzadeh.aspx" rel="external nofollow noopener" target="_blank">Ghazaleh Taherzadeh</a>,¬†<a href="https://scholar.google.com/citations?user=RVEFII8AAAAJ&amp;hl=en&amp;oi=ao" rel="external nofollow noopener" target="_blank">Alok Sharma</a>,¬†<b><u>Shubhashis Roy Dipta</u></b>,¬†<a href="https://cs.camden.rutgers.edu/faculty-staff/iman-dehzangi-ph-d/" rel="external nofollow noopener" target="_blank">Abdollah Dehzangi</a>,¬†and¬†<a href="https://scholar.google.com/citations?user=_D9NsFMAAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">Swakkhar Shatabda</a> </div> <div class="periodical"> <em>IEEE access</em>, Jul 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7751949/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <span class="__dimensions_badge_embed__" data-doi="10.1109/access.2020.2989713" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Post Translational Modification (PTM) is considered an important biological process with a tremendous impact on the function of proteins in both eukaryotes, and prokaryotes cells. During the past decades, a wide range of PTMs has been identified. Among them, malonylation is a recently identified PTM which plays a vital role in a wide range of biological interactions. Notwithstanding, this modification plays a potential role in energy metabolism in different species including Homo Sapiens. The identification of PTM sites using experimental methods is time-consuming and costly. Hence, there is a demand for introducing fast and cost-effective computational methods. In this study, we propose a new machine learning method, called Mal-Light, to address this problem. To build this model, we extract local evolutionary-based information according to the interaction of neighboring amino acids using a bi-peptide based method. We then use Light Gradient Boosting (LightGBM) as our classifier to predict malonylation sites. Our results demonstrate that Mal-Light is able to significantly improve malonylation site prediction performance compared to previous studies found in the literature. Using Mal-Light we achieve Matthew‚Äôs correlation coefficient (MCC) of 0.74 and 0.60, Accuracy of 86.66% and 79.51%, Sensitivity of 78.26% and 67.27%, and Specificity of 95.05% and 91.75%, for Homo Sapiens and Mus Musculus proteins, respectively. Mal-Light is implemented as an online predictor which is publicly available at: (http://brl.uiu.ac.bd/MalLight/).</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">ahmad2020mal</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Mal-light: Enhancing lysine malonylation sites prediction problem using evolutionary-based features}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Ahmad, Md Wakil and Arafat, Md Easin and Taherzadeh, Ghazaleh and Sharma, Alok and Dipta, Shubhashis Roy and Dehzangi, Abdollah and Shatabda, Swakkhar}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{IEEE access}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{8}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{77888--77902}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/access.2020.2989713}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col-sm-2 abbr"><abbr class="badge" style="background-color:#FF6600"><a href="https://www.elsevier.com/" rel="external nofollow noopener" target="_blank">Elsevier</a></abbr></div> <div id="dipta2020semal" class="col-sm-8"> <div class="title">SEMal: Accurate protein malonylation site predictor using structural and evolutionary information</div> <div class="author"> <b><u>Shubhashis Roy Dipta</u></b>,¬†<a href="https://www.wilkes.edu/campus-directory/ghazaleh.taherzadeh.aspx" rel="external nofollow noopener" target="_blank">Ghazaleh Taherzadeh</a>,¬†Md Wakil Ahmad,¬†Md Easin Arafat,¬†<a href="https://scholar.google.com/citations?user=_D9NsFMAAAAJ&amp;hl=en" rel="external nofollow noopener" target="_blank">Swakkhar Shatabda</a>,¬†and¬†<a href="https://cs.camden.rutgers.edu/faculty-staff/iman-dehzangi-ph-d/" rel="external nofollow noopener" target="_blank">Abdollah Dehzangi</a> </div> <div class="periodical"> <em>Computers in biology and medicine</em>, Jul 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S001048252030353X" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/dipta007/SEMal" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.compbiomed.2020.104022" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> </div> <div class="abstract hidden"> <p>Post Transactional Modification (PTM) is a vital process which plays an important role in a wide range of biological interactions. One of the most recently identified PTMs is Malonylation. It has been shown that Malonylation has an important impact on different biological pathways including glucose and fatty acid metabolism. Malonylation can be detected experimentally using mass spectrometry. However, this process is both costly and time-consuming which has inspired research to find more efficient and fast computational methods to solve this problem. This paper proposes a novel approach, called SEMal, to identify Malonylation sites in protein sequences. It uses both structural and evolutionary-based features to solve this problem. It also uses Rotation Forest (RoF) as its classification technique to predict Malonylation sites. To the best of our knowledge, our extracted features as well as our employed classifier have never been used for this problem. Compared to the previously proposed methods, SEMal outperforms them in all metrics such as sensitivity (0.94 and 0.89), accuracy (0.94 and 0.91), and Matthews correlation coefficient (0.88 and 0.82), for Homo Sapiens and Mus Musculus species, respectively. SEMal is publicly available as an online predictor at: http://brl.uiu.ac.bd/SEMal/.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">dipta2020semal</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SEMal: Accurate protein malonylation site predictor using structural and evolutionary information}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dipta, Shubhashis Roy and Taherzadeh, Ghazaleh and Ahmad, Md Wakil and Arafat, Md Easin and Shatabda, Swakkhar and Dehzangi, Abdollah}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computers in biology and medicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{125}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{104022}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.compbiomed.2020.104022}</span><span class="p">,</span>
  <span class="na">dimensions</span> <span class="p">=</span> <span class="s">{true}</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="sticky-bottom mt-5"> <div class="container" style="text-align: center;"> ¬© Copyright 2024 Shubhashis Roy Dipta. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script type="text/javascript">$(function(){$('[data-toggle="tooltip"]').tooltip()});</script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?7b30caa5023af4af8408a472dc4e1ebb"></script> <script defer src="https://unpkg.com/bootstrap-table@1.22.1/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?ec5242799e47c749ea0cb387fe27ae14"></script> <script defer src="/assets/js/common.js?45c116ad8084452442fc5feda110fc7f"></script> <script defer src="/assets/js/copy_code.js?29038c300c8185b066d52529c1328f2a" type="text/javascript"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script async src="https://www.googletagmanager.com/gtag/js?id=G-YM80KFF174"></script> <script>function gtag(){window.dataLayer.push(arguments)}window.dataLayer=window.dataLayer||[],gtag("js",new Date),gtag("config","G-YM80KFF174");</script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="https://unpkg.com/isotope-layout@3/dist/isotope.pkgd.min.js"></script> <script src="https://unpkg.com/imagesloaded@5/imagesloaded.pkgd.min.js"></script> <script>var $grid=$(".portfolio-grid").imagesLoaded(function(){$grid.isotope({getSortData:{importance:"[importance] parseInt"},itemSelector:".grid-item",percentPosition:!0,masonry:{columnWidth:".grid-sizer",gutter:".gutter-sizer",horizontalOrder:!0},filter:"*"})});$(".filter-button-group").on("click","a",function(){var t=$(this).attr("data-filter");$grid.isotope({filter:t,sortBy:"importance",sortAscending:!1})}),$(".button-group a.btn").on("click",function(){$(".button-group a.btn").removeClass("active"),$(this).addClass("active")});</script> <script src="/assets/js/github-stars.js"></script> <script type="text/javascript">githubStars("https://github.com/dipta007/cookbook-007",function(t){$("#dipta007-cookbook-007-stars").text(""+t)}),githubStars("https://github.com/dipta007/amazon-scrapper-1M-nodejs",function(t){$("#dipta007-amazon-scrapper-1M-nodejs-stars").text(""+t)}),githubStars("https://github.com/dipta007/Competitive-Programming",function(t){$("#dipta007-Competitive-Programming-stars").text(""+t)});</script> </body> </html>